vendor/github.com/gorilla/websocket/x_net_proxy.go

```diff
@@ -347,7 +349,7 @@ func (s *proxy_socks5) connect(conn net.Conn, target string) error {
 		return err
 	}
 
-	port, err := strconv.Atoi(portStr)
+	port, err := netutils.ParsePort(portStr, false)
```

Comment:
We can't hand-edit vendored code.  You probably have to drop all of these.  The correct fix is to fix them in their own upstreams, probably not worthwhile.